Without this patch, cargo fails to build in rust 1.20 due to:
error[E0004]: non-exhaustive patterns: `&__Nonexhaustive { }` not covered

--- src/cargo/util/errors.rs.orig
+++ src/cargo/util/errors.rs
@@ -83,7 +83,7 @@
             &CargoErrorKind::Parse(_) |
             &CargoErrorKind::Git(_) |
             &CargoErrorKind::Internal(_) |
-            &CargoErrorKind::CargoTestErrorKind(_) => false
+            &CargoErrorKind::CargoTestErrorKind(_) | _ => false
         }
     }
 }
